apps/wallet-mobile/src/i18n/getRegion.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Extracts the region segment from a locale or region string, or returns 'US' as fallback.
+ *
+ * @param {string} localeOrRegion - A string representing a locale or region, typically in the format of "language_region" or "language-region".
+ * @returns {string} The extracted region segment if found, otherwise returns 'US' as fallback.
+ * @example getRegion('en-US') // 'US'
+ * @example getRegion('pt_BR') // 'BR'
+ * @example getRegion('po') // 'US'
+ */
+export const getRegion = (localeOrRegion: string) => {
+  const match = /[_-](\w+)/.exec(localeOrRegion)
+  return match ? match[1] : 'US'
```

Comment:
not happy with this index accessing maybe `if (match && match.length > 2) const [,region] = match` maybe